docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,184 @@
+	<base href="../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+<body>
+	<h1>[name]</h1>
+
+	<h2>Overview</h2>
+	<div>
+		<p>
+			<strong>TL;DR</strong> three.js supports most modern browsers, including Internet Explorer 11 and above.</p>
+		<p>
+			three.js provides WebGL, Canvas, SVG and CSS renderers. Different renderers may have different requirements of compatibility.
+			In theory, three.js works on all browsers that support WebGL: Google Chrome 9+, Firefox 4+, Opera 15+, Safari
+			5.1+, Internet Explorer 11 and Microsoft Edge. More infomation can be found at
+			<a href="https://caniuse.com/#feat=webgl" target="_blank">Can I use WebGL?</a>.
+		</p>
+	</div>
+
+	<h2>Internet Explorer Compatibily</h2>
+	<div>
+		<p>
+			It is strongly recommended to upgrade older versions of IE to IE11. If you still need to support IE9, you can use CanvasRenderer
+			instead. Note that CanvasRenderer only supports a small subset of WebGLRenderer's features.
+		</p>
+		<p>
+			For Internet Explorer, the table below lists avaiable renderers.
+		</p>
+		<table>
+			<thead>
+				<tr>
+					<th>Renderer</th>
+					<th>IE Version</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>WebGLRenderer</td>
+					<td>IE11+</td>
+				</tr>
+				<tr>
+					<td>CanvasRenderer</td>
+					<td>IE9+</td>
+				</tr>
+				<tr>
+					<td>SVGRenderer</td>
+					<td>IE9+</td>
+				</tr>
+				<tr>
+					<td>CSS3DRenderer</td>
+					<td>IE10+</td>
+				</tr>
+				<tr>
+					<td>CSS2DRenderer</td>
+					<td>IE9+</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+
+	<h2>Part of ECMAScript Language Features or Web APIs Used in three.js</h2>
+	<div>
+		<p>
+			Here are some features used in three.js. Some of them may require additional polyfills.
+		</p>
+		<table>
+			<thead>
+				<tr>
+					<th>Feature</th>
+					<th>Use Scope</th>
+					<th>Modules</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Typed Arrays</td>
+					<td>Source</td>
+					<td>BufferAttribute, BufferGeometry, etc.</td>
+				</tr>
+				<tr>
+					<td>Web Audio API</td>
+					<td>Source</td>
+					<td>Audio, AudioContext, AudioListener, etc.</td>
+				</tr>
+				<tr>
+					<td>WebVR API</td>
+					<td>Source</td>
+					<td>WebVRManager, etc.</td>
+				</tr>
+				<tr>
+					<td>Blob</td>
+					<td>Source</td>
+					<td>FileLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>performance</td>
+					<td>Source</td>
+					<td>Clock, etc.</td>
+				</tr>
+				<tr>
+					<td>requestAnimationFrame</td>
+					<td>Source</td>
+					<td>WebGLRenderer, VideoTexture, etc.</td>
+				</tr>
+				<tr>
+					<td>Promise</td>
+					<td>Examples</td>
+					<td>GLTFLoader, GLTF2Loader, WEBVR, etc.</td>
+				</tr>
+				<tr>
+					<td>Fetch</td>
+					<td>Examples</td>
+					<td>ImageBitmapLoader, etc.</td>
+				</tr>
+				<tr>
+					<td>Web Workers</td>
+					<td>Examples</td>
+					<td>WWOBJLoader2, etc.</td>
```

Comment:
I think WWOBJLoader2 is now OBJLoader2 and uses workers only if they are available, so you can leave this on out.